### PR TITLE
sci-electronics/kicad: Fix missing footprint library issue

### PR DESCRIPTION
**Pull request by zpuskas**
Gentoo Bug: 576820

Package-Manager: portage-2.2.28

### CODE REVIEW

**Comment by zpuskas** [comment]
cc: @idella 
cc: @jlec 
cc: @proxy-maint

**Comment by zpuskas** [comment]
Migrating to EAPI=6 is not possible for now as it is blocked on wxwidgets.eclass. Other issues mentioned @jlec last time were fixed in the ebuilds.

**Comment by monsieurp** [comment]
@gentoo/proxy-maint

**Comment by idella** [comment]
Zoltan can you expand the commit msg to record all that was in fact done;
revbump of 3 individual versions + adding the file kicad-footprints-cmakelists.txt to "files/"
The log record ought inform of all notable changes of this type. Then it should be fine

File: sci-electronics/kicad/kicad-4.0.0-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,191 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
```
Comment (jlec) [inline sci-electronics/kicad/kicad-4.0.0-r1.ebuild]: @zpuskas Any reason why not using EAPI=6?

**Pull request closed without merging**